File: Python/basicMath.py
inputName = input("What's your name? ")
if inputName == "":
	inputName = "Jakob"

inputVar1 = input(f"Type in a number, {inputName} ")

inputVar2 = input(f"Type in another number, {inputName} ")

inputSum = input(f"Now add {inputVar1} and {inputVar2} = ")

if int(inputSum) == int(inputVar1) + int(inputVar2):
	print(f"Nice one {inputName}!")
else:
	print(f"Incorrect answer {inputName} !")
	print(f"Correct answer is: {int(inputVar2) + int(inputVar1)}")

File: Lua/basicMath.lua
-- https://stackoverflow.com/questions/12069109/getting-input-from-the-user-in-lua
-- https://www.lua.org/pil/4.3.1.html

io.write("What's you name? ")
inputName = io.read()
if inputName == "" then
	inputName = "Jakob"
end

io.write("Type in a number, " .. inputName .. " ")
inputVar1 = io.read("*n")
io.write("Type in another number, " .. inputName .. " ")
inputVar2 = io.read("*n")
io.write("Now add " .. inputVar1 .. " and " .. inputVar2 .." = ")
inputSum = io.read("*n")
if inputSum == inputVar2 + inputVar1 then
	print("Good job " .. inputName .. "!")
else
	print("Incorrect answer " .. inputName .. "!")
	print("Correct answer is: " .. (inputVar2 + inputVar1))
end